src/Pact/Gas/Table.hs

```diff
@@ -0,0 +1,173 @@
+  , _gasCostConfig_functionApplicationCost = 1
+  }
+
+testGasTable :: Map Text ([Term Ref] -> Gas)
+testGasTable =
+  let lookupCost = 10
+  in Map.fromList
+    [ ("describe-module", const 100)
+    , ("sort", const 100)
+    , ("keys", const 1)
+    , ("map", const 1)
+    , ("!=", const 1)
+    , ("with-capability",const 1)
+    , ("create-pact-guard", const 1)
+    , ("/", const 1)
+    , ("ln", const 1)
+    , ("enforce-pact-version", const 1)
+    , ("or?", const 1)
+    , ("txids", const 1)
+    , ("or", const 1)
+    , ("yield", const 1)
+    , ("diff-time", const 1)
+    , ("minutes", const 1)
+    , ("make-list", const 1)
+    , ("reverse", const 1)
+    , ("filter", const 1)
+    , ("select", const 1)
+    , ("compose", const 1)
+    , ("contains", const 1)
+    , ("with-default-read", const 1)
+    , ("create-module-guard", const 1)
+    , (">=", const 1)
+    , ("keys-2", const 1)
+    , ("take", const 1)
+    , ("format-time", const 1)
+    , ("constantly", const 1)
+    , ("enforce-keyset", const 1)
+    , ("identity", const 1)
+    , ("describe-table", const 1)
+    , ("enforce", const 1)
+    , ("pact-id", const 1)
+    , ("keyset-ref-guard", const 1)
+    , ("add-time", const 1)
+    , ("update", const 1)
+    , ("read-integer", const 1)
+    , ("resume", const 1)
+    , ("typeof", const 1)
+    , ("pact-version", const 1)
+    , ("and?", const 1)
+    , ("log", const 1)
+    , ("read-decimal", const 1)
+    , ("length", const 1)
+    , ("sqrt", const 1)
+    , ("<", const 1)
+    , ("days", const 1)
+    , ("create-table", const 1)
+    , ("keys-any", const 1)
+    , ("insert", const 1)
+    , ("if", const 1)
+    , ("enforce-one", const 1)
+    , ("*", const 1)
+    , ("not?", const 1)
+    , ("read-keyset", const 1)
+    , ("read", const 1)
+    , ("list", const 1)
+    , ("round", const 1)
+    , ("hash", const 1)
+    , ("exp", const 1)
+    , ("not", const 1)
+    , ("at", const lookupCost)
+    , ("where", const 1)
+    , ("-", const 1)
+    , ("str-to-int", const 1)
+    , ("=", const 1)
+    , ("describe-keyset", const 1)
+    , ("time", const 1)
+    , ("with-read", const 1)
+    , ("write", const 1)
+    , ("define-keyset", const 1)
+    , ("parse-time", const 1)
+    , ("create-user-guard", const 1)
+    , ("floor", const 1)
+    , ("abs", const 1)
+    , ("keys-all", const 1)
+    , ("txlog", const 1)
+    , ("require-capability", const 1)
+    , ("keylog", const 1)
+    , ("ceiling", const 1)
+    , ("hours", const 1)
+    , ("+", const 1)
+    , ("format", const 1)
+    , ("and", const 1)
+    , ("tx-hash", const 1)
+    , (">", const 1)
+    , ("enforce-guard", const 1)
+    , ("mod", const 1)
+    , ("remove", const 1)
+    , ("^", const 1)
+    , ("list-modules", const 1)
+    , ("read-msg", const 1)
+    , ("fold", const 1)
+    , ("bind", const 1)
+    , ("<=", const 1)
+    , ("drop", const 1)
+    ]
+
+tableGasModel :: GasCostConfig -> GasModel
+tableGasModel gasConfig = GasModel $ \name -> \case
```

Comment:
Sorry, but I pretty much hate `LambdaCase`, and here it just looks repulsive with the point-free matching on the the second argument. Please don't.